tests/Transports.Subscriptions.WebSockets.Tests/WebSocketsConnectionFacts.cs

```diff
@@ -39,10 +39,6 @@ public WebSocketsConnectionFacts()
                 .Start();
 
             _server = _host.GetTestServer();
-
-            // Workaround until GraphQL can swap off Newtonsoft.Json and onto the new MS one.
-            // https://github.com/graphql-dotnet/graphql-dotnet/issues/1116
-            _server.AllowSynchronousIO = true;
```

Comment:
I see that this project refers `GraphQL.NewtonsoftJson`. How does it work now without ` _server.AllowSynchronousIO = true;` ?

Reply:
I wondered this too, and essentially the methods used in the web sockets middleware are async. `AcceptWebSocketAsync`.

I'm not totally across how all this works, but for instance `GraphQLWebSocketsMiddleware` does:
* `AcceptWebSocketAsync` then hooks up a connection
* `CloseAsync`

`WebSocketReaderPipeline` in `ReadMessageAsync` looks all async.

`WebSocketWriterPipeline` hmm, that uses `IDocumentWriter`'s `WriteAsync` to write onto a `WebsocketWriterStream` but that isn't directly the http response stream, there's tonnes of websocket abstractions in there, so... honestly. Not sure. Will quickly run it up in a UI and check it's all working too.

Reply:
Yea seems to be fine. I'm happy to leave this to the other issue I raised.

Reply:
Actually, the `IDocumentWriter` registered is actually the new S.T.Json one, so that's why the response writing is probably OK. And like I said, the reading is done not straight into a JSON object, so it's fully read async first and that is OK.

Reply:
ok